odhargo/cluster/base/cronworkflow-crd.yaml

```diff
@@ -1,15 +0,0 @@
-# This is an auto-generated file. DO NOT EDIT
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: cronworkflows.argoproj.io
-spec:
-  group: argoproj.io
-  names:
-    kind: CronWorkflow
-    plural: cronworkflows
-    shortNames:
-    - cronwf
-    - cwf
-  scope: Namespaced
-  version: v1alpha1
```

Comment:
Does the new version of argo not support cronworkflows?

Reply:
Yes it does, but it's relocated to [`cluster/base/crds.yaml`](https://github.com/opendatahub-io/odh-manifests/blob/2cc43411f49166dcb98ffb1b239d9c14c5efce39/odhargo/cluster/base/crds.yaml) since this file can be generated from the upstream manifests by the auto updater script and it would include all the CRDs at once, so we can't forget about any.